spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -4,6 +4,11 @@
   let(:mascot_account)             { create(:user) }
   let!(:welcome_thread)            { create(:article, user: mascot_account, published: true, tags: "welcome") }
   let!(:welcome_broadcast)         { create(:welcome_broadcast) }
+  let!(:twitter_connect_broadcast) { create(:twitter_connect_broadcast) }
+  let!(:github_connect_broadcast)  { create(:github_connect_broadcast) }
+  let!(:customize_feed_broadcast)  { create(:customize_feed_broadcast) }
+  let!(:discuss_and_ask_broadcast) { create(:discuss_and_ask_broadcast) }
+  let!(:customize_ux_broadcast)    { create(:customize_ux_broadcast) }
```

Comment:
do broadcast need to be recreated each time per each test or they can just sit there in the DB with `let_it_be_readonly` ?

Reply:
Oh, I completely forgot about this. Thanks for pointing this out!